server/app/mutations/stacks/patch.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'common'
+
+module Stacks
+  class Patch < Mutations::Command
+    include Common
+
+    required do
+      model :stack_instance, class: Stack
+    end
+
+    optional do
+      model :grid, class: Grid
```

Comment:
Doesn't need a `grid` parameter.

(not for this PR, but this should also get cleaned up from the `Stacks::Update` as well - it's only used for the `validate_volumes`, and should get passed as a parameter instead).

Reply:
✅ RESOLVED